webapp/templates/sidebar.html.tmpl

```diff
@@ -26,10 +26,10 @@
   <footer class="footer-side-menu">
     <ul class="footer-list">
       <li class="footer-list-item">
-        <i class="eos-icons" title="" data-html="true" data-toggle="tooltip" data-title="{{ escapedTemplate "license" . }}" data-trigger="hover click">find_in_page</i>
+        <i class="eos-icons" title="" data-html="true" data-toggle="tooltip" data-title="SUSE" data-trigger="hover click">find_in_page</i>
```

Comment:
this is a shortcut i took, because of the `escapedTemplate` caused panic

Reply:
We introduced that template function to render escaped HTML code inside HTML attributes.

Reply:
done